README.md

```diff
@@ -205,7 +205,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
 * `analyse` - Run a Codacy analysis over a directory/files
     * `--verbose` - Run the tool with verbose output
     * `--tool` - Choose the tool to analyse the code (e.g. brakeman)
-    * `--directory` - Choose the directory to be analysed
+    * `--directory` - Absolute path to the directory to be analysed
```

Comment:
The absolute path only seems to be required when running codacy-analysis-cli in a Docker container. So my suggestion is to mention this under the "Docker" section and not here. :thinking:

Reply:
**Another update:** The absolute path is never required after all, and the issue was caused by something else. :raised_hands: